backend/sprint/sprint/settings.py

```diff
@@ -85,8 +85,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': os.environ['DATABASE_NAME'],
+        'USER': os.environ['DATABASE_USERNAME'],
+        'PASSWORD': os.environ['DATABASE_PASSWORD'],
+        'HOST': os.environ['DATABASE_URL'],
+        'PORT': os.environ['DATABASE_PORT'],
```

Comment:
Could you please update the notion document for environment variables and share the link in the description? It would be better to add some summary about your change to the description. Thanks for your DB update!

Reply:
Okay. Thank you :)

Reply:
QQ) Does this repository have the `env.sh` file? or Do we have to create it locally?

Reply:
You have to create or download it from notion documentation.